keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -419,10 +442,16 @@ def _backward(self, y_true, y_pred):
 
         self.classification_loss_metric.update_state(classification_loss)
```

Comment:
unrelated to this PR -- consider refactoring RetinaNet, for example, `compute_loss` called in `_backward` is weird, as it's part of forward path.

Reply:
Good point, that is a really bad name.  I'll have to update it.  Thanks!